README.md

```diff
@@ -72,6 +72,22 @@ Packaging for your favorite distribution would be a welcome contribution!
 
 ## Compiling Monero from Source
 
+### Docker
```

Comment:
Thanks for the docker script. Could you please move this section as an item in the list of "Installing Monero from Package"? Docker fits into the same category as a PKGBUILD script (AUR) or a homebrew script (OS X).